From bef9993c4ca53a4d9e1b98b9882858d67940ab94 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Wed, 7 Aug 2013 18:55:49 +0000
Subject: [PATCH 117/462] GPPDevCustomLib.lib status monitoring library for
 custom printer V1.0.0.0 Mantis 0028323: GPPDev: Custom VKP-80II

LIB_DST_GPPDEV_CUSTOM  = $(_LIB)\GPPDevCustom.lib
LIB_GPPDEV_CUSTOM      = $(LIB_DST_GPPDEV_CUSTOM)


git-svn-id: svn://localhost/SelfServiceBuild/trunk@170 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.mak | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SelfServiceBuild/Build/cpp/makefile.mak b/SelfServiceBuild/Build/cpp/makefile.mak
index 2f6b3aa..8ec3613 100644
--- a/SelfServiceBuild/Build/cpp/makefile.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.mak
@@ -116,6 +116,7 @@ PLATFORM_DIRECTORIES= \
  Massai\cpp\Contxt \
  Massai\cpp\Conveyor \
  Massai\cpp\ConveyorTest \
+ Massai\cpp\GPPDevCustomLib \
  Massai\cpp\GPPDev \
  Massai\cpp\GPPDev\windows_IM63 \
  Massai\cpp\ScreenResolution \
-- 
2.41.0.windows.1

